parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,99 @@
+# Parking Lot Help Request Firmware
+
+This project is meant to be host-free because the Notecard itself can be configured to send notes to Notehub when the help button is pressed.
+
+Instead of a true firmware set up guide, the [first part](#configure-the-notecard) of this document will guide you through creating a reusable Notecard configuration script which can be executed through the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/) or through the [online webREPL](https://dev.blues.io/notecard-playground/) available on the Blues Developer Experience site.
+
+The [second section](#transform-notehub-data-with-jsonata-for-twilio), covers the data transformation that will need to happen inside of a Notehub project before a button press event sent to Notehub is routed out to Twilio to trigger an SMS alert to a user's phone.
```

Comment:
Suggested rewording you can take or leave ⬇️ 

The section section covers a data transformation you need to provide in your Notehub project to triggers SMS alerts.

Reply:
Thank you, I tend to get too verbose after a while. Much simpler